src/css.rs

```diff
@@ -98,21 +105,28 @@ button.grayed-out {
 button.white-score {
     background-color: #FFFFFF;
     color: #000000;
+    border-top-right-radius: 0px;
+    border-top-left-radius: 0px;
```

Comment:
Nice, much easier and cleaner solution than what I had in mind. However, probably should have a matching style to round the top corners of the labels.

Reply:
ok, I will look into that

Reply:
added rounded corners to the tops of the labels